src/components/hint/index.tsx

```diff
@@ -418,12 +472,11 @@ class Hint extends BaseComponent {
       <React.Fragment>
         {onBackgroundPress ? (
           <Modal
-            pointerEvents="box-none"
```

Comment:
Why are you removing the `pointerEvents`?

Reply:
`pointerEvents` is a `View`, not `Modal` prop.
https://reactnative.dev/docs/view#pointerevents